src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/CloudProvider.java

```diff
@@ -65,4 +65,6 @@
     public SimpleImmutableEntry<String, String> createKeyPair(Infrastructure infrastructure, Instance instance);
 
     public void deleteKeyPair(Infrastructure infrastructure, String keyPairName, String region);
+
+    public Set<NodeCandidate> getNodeCandidate(Infrastructure infra, String region, String imageReq);
```

Comment:
A comment with some examples for `imageReq` to show how to use it ? The comment could be done in each cloud impl when it's cloud-specific.

Reply:
I have just added a generic explanation for this feature in the Javadoc of this interface.